tests/functional/registry/test_config.py

```diff
@@ -55,3 +55,19 @@ def test_banksy(gov, registry, create_vault, rando):
 
     assert not registry.banksy(gov)
     registry.tagVault(vault, "", {"from": gov})
+
+
+def test_governance(guardian, rewards, registry, create_token, create_vault, rando):
+    # Not just anyone can create a new endorsed Vault, only governance can!
+    with brownie.reverts():
+        registry.newVault(create_token(), guardian, rewards, "", "", {"from": rando})
+
+    vault = create_vault()
+
+    # Not just anyone can create a new Release either
+    with brownie.reverts():
+        registry.newRelease(vault, {"from": rando})
+
+    # Not just anyone can endorse a Vault either
+    with brownie.reverts():
+        registry.endorseVault(vault, {"from": rando})
```

Comment:
@fubuloubu I moved the tests to their own function, so they aren't conflated with the banksy tests. Is this what you had in mind?

Reply:
Add this to https://github.com/iearn-finance/yearn-vaults/blob/master/tests/functional/registry/test_deployment.py

Reply:
moved